nsv13/code/modules/jobs/security/weapons.dm

```diff
@@ -61,28 +61,28 @@
 	desc = "A handgun that's never let its owner down before. It's got a pleasant wooden grip with plenty of detailing etched into it. A nice, all round weapon to defend yourself with."
 
 /datum/design/rubbershot
-	name = "9mm Glock Round"
+	name = "9mm rubber Glock round"
 	id = "glock_ammo"
 	build_type = AUTOLATHE
 	materials = list(/datum/material/iron = 500)
 	build_path = /obj/item/ammo_casing/c9mm/rubber
 	category = list("initial", "Security")
 
 /datum/design/tazer
-	name = "Tazer Round"
+	name = "3mm electro-shock tazer round"
```

Comment:
TASER.

Reply:
Not gonna change it for one thing if all the other mentions of it say tazer, but I might edit all of those later.